chaste_codegen/data/tests/chaste_reference_models/RLopt/dynamic_bondarenko_model_2004_apex.cpp

```diff
@@ -102,9 +102,9 @@
         const double var_non_inactivating_steady_state_potassium_current__i_Kss = 0.050000000000000003 * (-var_fast_transient_outward_potassium_current__E_K + var_chaste_interface__membrane__V) * var_chaste_interface__non_inactivating_steady_state_potassium_current__aKss * var_chaste_interface__non_inactivating_steady_state_potassium_current__iKss; // picoA_per_picoF
         const double var_slow_delayed_rectifier_potassium_current__i_Ks = 0.0057499999999999999 * pow(var_chaste_interface__slow_delayed_rectifier_potassium_current__nKs, 2) * (-var_fast_transient_outward_potassium_current__E_K + var_chaste_interface__membrane__V); // picoA_per_picoF
         const double var_slow_transient_outward_potassium_current__i_Kto_s = 0; // picoA_per_picoF
-        const double var_fast_sodium_current__E_Na = 25.674321243523316 * log(140600.0 / (0.1111111111111111 * var_chaste_interface__potassium_concentration__Ki + var_chaste_interface__sodium_concentration__Nai)); // millivolt
+        const double var_fast_sodium_current__E_Na = 25.674321243523316 * log(126540.0 / (0.10000000000000001 * var_chaste_interface__potassium_concentration__Ki + 0.90000000000000002 * var_chaste_interface__sodium_concentration__Nai)); // millivolt
```

Comment:
Why have these changed?

Reply:
Seems to be the version in the master was a side effect of the 1.0* optimisation